<?xml version="1.0"?>

<launch>
    <node
        pkg="ro_slam_with_uwb" type="beacon_writer.py" name="beacon_writer_node"
        output="screen">

        <param name="csv_file_name" value="/home/albert/catkin_ws/src/ro_slam_with_uwb/scripts/test" />
        <param name="csv_file_mode" value="a" />
        <param name="csv_file_max" value="10" />
        <param name="beacon_topic_name" value="beacon_raw" />

        <!-- Triangle: -->
        <rosparam param="range_ground_truth">[
            176, 177, 1.0, 177, 178, 1.0, 178, 176, 1.0
        ]</rosparam>
        <!-- -->

        <!-- Pentagram:
        <rosparam param="range_ground_truth">[
            176, 177, 1.0, 177, 178, 1.0, 178, 179, 1.0, 179, 180, 1.0, 180, 176, 1.0
            176, 178, 1.0, 178, 180, 1.0, 180, 177, 1.0, 177, 179, 1.0, 179, 176, 1.0
        ]</rosparam>
        -->
    </node>
</launch>